zmq/tests/test_message.py

```diff
@@ -310,14 +305,14 @@ def test_noncopying_recv(self):
             sb.send(null, copy=False)
             m = sa.recv(copy=False)
             mb = m.bytes
-            # buf = view(m)
+            # buf = memoryview(m)
             buf = m.buffer
             del m
             for i in range(5):
                 ff=b'\xff'*(40 + i*10)
                 sb.send(ff, copy=False)
                 m2 = sa.recv(copy=False)
-                if view.__name__ == 'buffer':
+                if memoryview.__name__ == 'buffer':
```

Comment:
Same here - this will never be true

Reply:
I wrote on that commit: _There is opportunity to remove tests but let's make sure that these changes pass the current tests before removing any of them._